plugins/chrome-ux-report/src/api.test.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2021 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Config } from '@backstage/config';
+import { UrlPatternDiscovery } from '@backstage/core';
+import { ChromeUXReportClient } from './api';
+import nock from 'nock';
```

Comment:
we use `msw` everywhere else and are standardizing on that - would you mind checking around the codebase for examples of usages of that? it's pretty powerful and nice